properties-workitem/src/main/resources/META-INF/PropertiesWorkItemDefinition.wid

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
This file is not needed. The wid is generated from your wid annotations in the handler. Please remove.